code/modules/admin/admin_verbs.dm

```diff
@@ -26,7 +26,7 @@ GLOBAL_PROTECT(admin_verbs_admin)
 	return list(
 	/client/proc/list_pretty_filters,			//yogs - shows all the pretty filters
 	/client/proc/invisimin,				/*allows our mob to go invisible/visible*/
-//	/datum/admins/proc/show_traitor_panel,	/*interface which shows a mob's mind*/ -Removed due to rare practical use. Moved to debug verbs ~Errorage
+	/datum/admins/proc/show_traitor_panel,	/*interface which shows a mob's mind*/ -Removed due to rare practical use. Moved to debug verbs ~Errorage
```

Comment:
replace that /* multiline comment */ with a //line one, that part after the dash is not commented out